android/app/src/main/java/com/hotwirerndemo/MainApplication.java

```diff
@@ -24,11 +25,12 @@ public boolean getUseDeveloperSupport() {
 
         @Override
         protected List<ReactPackage> getPackages() {
-          @SuppressWarnings("UnnecessaryLocalVariable")
-          List<ReactPackage> packages = new PackageList(this).getPackages();
-          // Packages that cannot be autolinked yet can be added manually here, for example:
-          // packages.add(new MyReactNativePackage());
-          return packages;
+            @SuppressWarnings("UnnecessaryLocalVariable")
+            List<ReactPackage> packages = new PackageList(this).getPackages();
+            // Packages that cannot be autolinked yet can be added manually here, for example:
+            // packages.add(new MyReactNativePackage());
+            packages.add(new ReactAppPackage());
+            return packages;
         }
```

Comment:
Could be reformatted so the actual change is more visible in the diff (ie the `packages.add(new ReactAppPackage());` line only)